jobQueue.py

```diff
@@ -34,8 +34,29 @@
 class JobQueue(object):
 
     def __init__(self, preallocator):
+        """
+        Here we maintain several data structures used to keep track of the 
+        jobs present for the autograder. 
+
+        Live jobs contains:
+        - jobs that are yet to be assigned and run
+        - jobs that are currently running
+
+        Dead jobs contains: 
+        - jobs that have been completed, or have been 'deleted' when in
+          the live jobs queue
+
+        This is a FIFO queue of jobs that are pending assignment. 
```

Comment:
I think you wanted to mention unassignedJobs here

Reply:
pog